drgn_tools/targetcli.py

```diff
@@ -49,6 +49,20 @@ def for_each_iscsi_tpg(tiqn: Object) -> Iterable[Object]:
     )
 
 
+def for_each_portal(tpg: Object) -> Iterable[Object]:
+    """
+    Get a list of portals under tpg
+
+    :param tpg: ``struct se_portal_group``
+    :returns:  Iterator of str
+    """
+    np_dentry = tpg.tpg_np_group.cg_item.ci_dentry
+    for portal in list_for_each_entry(
+        "struct dentry", np_dentry.d_subdirs.address_of_(), "d_child"
+    ):
```

Comment:
Please use `dentry_for_each_child()` from `drgn_tools/dentry.py` which handles the new changes in `struct dentry` in Linux 6.8.